README.adoc

```diff
@@ -61,6 +61,19 @@ assert_equal(nil, actual)
 assert_nil(actual)
 ----
 
+=== Refute Nil [[refute-nil]]
+
+Use `refute_nil` if not expecting `nil`.
+
+[source,ruby]
+----
+# bad
+assert(actual.nil?)
```

Comment:
This does not seem to be an expected example because the result is different from good case.

Reply:
My bad. I'll fix it in some time. It should be `refute(actual.nil?)` or `assert(!actual.nil?)`